modules/backend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@wintercms/wn-backend-module",
+    "version": "1.0.0",
+    "description": "The Backend module for Winter CMS.",
+    "private": true,
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/wintercms/winter.git"
+    },
+    "author": {
+        "name": "Ben Thomson",
+        "email": "git@alfreido.com",
+        "url": "https://wintercms.com"
+    },
+    "contributors": [
+        {
+            "name": "Winter CMS Maintainers",
+            "url": "https://wintercms.com"
+        }
+    ],
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/wintercms/winter/issues"
+    },
+    "homepage": "https://wintercms.com/",
+    "dependencies": {
+        "@popperjs/core": "^2.11.4",
+        "js-cookie": "^3.0.1",
+        "vue": "^3.2.31"
```

Comment:
Is Vue being loaded for the backend right now?

Reply:
It's not, it's not even included in the build since we're not using it. I'll remove it - I think I borrowed this config from the Inspector rewrite which is using Vue.